onnxruntime/core/providers/cpu/tensor/quantize_linear.h

```diff
@@ -13,27 +13,29 @@ template <typename T>
 class DequantizeLinear final : public OpKernel {
  public:
   DequantizeLinear(const OpKernelInfo& info) : OpKernel(info) {
-    has_axis_ = info.GetAttr<int64_t>("axis", &axis_).IsOK();
+    if (!info.GetAttr<int64_t>("axis", &axis_).IsOK()) {
+      axis_ = 1;
+    }
   }
 
   Status Compute(OpKernelContext* context) const override;
 
-  private:
+ private:
   int64_t axis_ = 0;
```

Comment:
does axis_ need to be initialized here given how the constructor either reads an axis or sets the value to -1 (same in QuantizeLinear)?